multiqc/modules/dedup/dedup.py

```diff
@@ -104,6 +108,7 @@ def dedup_alignment_plot (self):
         config = {
             'id': 'dedup_rates',
             'title': 'DeDup: Deduplicated Reads',
+            'description': 'Read categories that were either not removed (= unique reads) or removed (= duplicates).'
```

Comment:
Same thing - good description but in the wrong place. I'll move it in a second 👍

Reply:
Ignore me sorry - this is in a table, so it _is_ correct. Ok, pretty confusing! 😕

Reply:
No wait! It's a plot! So the first comment was correct!

Ok, deleting comments and getting a coffee sorry..